src/WebJobs.Extensions.DurableTask/DurableTaskOptions.cs

```diff
@@ -97,6 +97,32 @@ public class DurableTaskOptions
         /// <value>The name of a connection string that exists in the app's application settings.</value>
         public string AzureStorageConnectionStringName { get; set; }
 
```

Comment:
Do we need Validation for this class for kind error message related ConnectionString? 
What do you think?

Reply:
What kind of validation?

Reply:
DurableTaskOptions.cs has a validate method for the property.  For example, we can validate if it is configured(not null). Then people might realize that the ConnectionString has not set.  However, it is  not mandatory. So I ask you the question.